class UserAlreadyExistsException(Exception):
    pass


class UserDoesNotExistException(Exception):
    pass
